src/main/java/attendance/AttendanceProgram.java

```diff
@@ -0,0 +1,21 @@
+package attendance;
+
+import attendance.controller.AttendanceController;
+import attendance.view.InputView;
+import attendance.view.OutputView;
+import java.time.LocalDateTime;
+
+public class AttendanceProgram {
+
+    public static void main(String[] args) {
+        initializeAttendanceController().startAttendanceSystem(initializeBaseTime());
+    }
+
+    private static AttendanceController initializeAttendanceController() {
+        return new AttendanceController(new InputView(), new OutputView());
+    }
+
+    private static LocalDateTime initializeBaseTime() {
+        return LocalDateTime.now();
+    }
```

Comment:
```suggestion
    private static LocalDateTime initializeBaseTime() {
        return LocalDateTime.of(2024,12,13, 10,1);
    }
```

- 특정 날짜가 아닌 새롭게 지정한 날짜에도 동작할 수 있도록 분리했었습니다.
  - 출력 예시처럼 하려면, 아래의 날짜를 12월 13일로 바꿔주시면 됩니다.